src/connection_wrap.h

```diff
@@ -0,0 +1,22 @@
+#ifndef SRC_CONNECTION_WRAP_H_
+#define SRC_CONNECTION_WRAP_H_
+
+#if defined(NODE_WANT_INTERNALS) && NODE_WANT_INTERNALS
+
+#include "env.h"
+#include "v8.h"
+
+namespace node {
+
+class ConnectionWrap {
+ protected:
+  template<typename WrapType, typename UVType>
```

Comment:
Ah, that does sound better. Let me give that a try and see what you think. Thanks